src/ascii/chain/setting/private/v6_14.rs

```diff
@@ -0,0 +1,362 @@
+    /// The type of the [`limit.home.pos`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.pos) setting.
+    pub struct LimitHomePos: Setting<Type = i64, Name = "limit.home.pos">, AxisScope;
+    /// The type of the [`limit.home.posupdate`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.posupdate) setting.
+    pub struct LimitHomePosupdate: Setting<Type = u8, Name = "limit.home.posupdate">, AxisScope;
+    /// The type of the [`limit.home.preset`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.preset) setting.
+    pub struct LimitHomePreset: Setting<Type = i64, Name = "limit.home.preset">, AxisScope;
+    /// The type of the [`limit.home.state`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.state) setting.
+    pub struct LimitHomeState: Setting<Type = bool, Name = "limit.home.state">, AxisScope;
+    /// The type of the [`limit.home.triggered`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.triggered) setting.
+    pub struct LimitHomeTriggered: Setting<Type = bool, Name = "limit.home.triggered">, AxisScope;
+    /// The type of the [`limit.home.type`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.home.type) setting.
+    pub struct LimitHomeType: Setting<Type = u8, Name = "limit.home.type">, AxisScope;
+    /// The type of the [`limit.max`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.max) setting.
+    pub struct LimitMax: Setting<Type = i64, Name = "limit.max">, AxisScope;
+    /// The type of the [`limit.min`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.min) setting.
+    pub struct LimitMin: Setting<Type = i64, Name = "limit.min">, AxisScope;
+    /// The type of the [`limit.swapinputs`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#limit.swapinputs) setting.
+    pub struct LimitSwapinputs: Setting<Type = bool, Name = "limit.swapinputs">, AxisScope;
+    /// The type of the [`maxspeed`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#maxspeed) setting.
+    pub struct Maxspeed: Setting<Type = u64, Name = "maxspeed">, AxisScope;
+    /// The type of the [`motion.accelonly`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#motion.accelonly) setting.
+    pub struct MotionAccelonly: Setting<Type = u32, Name = "motion.accelonly">, AxisScope;
+    /// The type of the [`motion.decelonly`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#motion.decelonly) setting.
+    pub struct MotionDecelonly: Setting<Type = u32, Name = "motion.decelonly">, AxisScope;
+    /// The type of the [`peripheralid`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#peripheralid) setting.
+    pub struct Peripheralid: Setting<Type = u32, Name = "peripheralid">, AxisScope;
+    /// The type of the [`pos`](https://www.zaber.com/w/Manuals/ASCII_Protocol_Manual#pos) setting.
+    pub struct Pos: Setting<Type = i64, Name = "pos">, AxisScope;
```

Comment:
It looks like the database used to generate this data has incorrect type information for some settings. The script is generating data correctly, but the database is incorrect. Get a fixed version before merging this.

Reply:
The types are fixed now.